src/Haskell/Ide/Engine/Plugin/ApplyRefact.hs

```diff
@@ -41,12 +41,9 @@ type HintTitle = T.Text
 applyRefactDescriptor :: PluginId -> PluginDescriptor
 applyRefactDescriptor plId = PluginDescriptor
   { pluginId = plId
-  , pluginName = "ApplyRefact"
-  , pluginDesc = "apply-refact applies refactorings specified by the refact package. It is currently integrated into hlint to enable the automatic application of suggestions."
   , pluginCommands =
-      [ PluginCommand "applyOne" "Apply a single hint" applyOneCmd
-      , PluginCommand "applyAll" "Apply all hints to the file" applyAllCmd
-      , PluginCommand "lint" "Run hlint on the file to generate hints" lintCmd
```

Comment:
Why is "lint" removed?

Reply:
It can't be accessed by LSP. Previously old IDEs communicating via the JSON transport would call this to get back a list of diagnostics. Now we just manually call it to send the publish diagnostic notifications in Server.hs

Reply:
Is it still possible to send a code action that maps back to this? or to any of the others that have been removed?

In particular, when `HaRe` is re-enabled, it sends commands which need to be activated when returned.

Reply:
The applyOne/applyAll ones are still there which hlint code actions can call, but as far as I can tell the lint command doesn’t really work with LSP since the only thing commands can do are return document edits?

Reply:
Ok, I guess we then have the concept visible, as actually used. So the currently unused ones can go.

Reply:
Yes, and the actual functionality is still there in the lint command which is just called manually